source/cmake_modules/CompilationFlags.cmake

```diff
@@ -76,7 +76,7 @@ if(STRIP_SYMBOLS)
     set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -s")
 endif(STRIP_SYMBOLS)
 
-set(NAVITIA_ALLOCATOR "tcmalloc")
+set(NAVITIA_ALLOCATOR "")
```

Comment:
If that's made to last, shouldn't we make it configurable, or dependent on platform?